dart/lib/src/hub.dart

```diff
@@ -225,6 +225,13 @@ class Hub {
       );
       return;
     }
+    if (userFeedback.name == null || (userFeedback.name?.isEmpty ?? true)) {
+      _options.logger(
+        SentryLevel.warning,
+        'Name is empty, dropping the feedback',
+      );
+      return;
+    }
```

Comment:
We can't do that because `name` isn't mandatory, at least 1 field is mandatory, either `name`, `email` or `comments`.